views/vm/index.erb

```diff
@@ -75,9 +75,11 @@
     locals: {
       icon: "hero-server-stack",
       title: "No virtual machines",
+      description: "You don't have permission to create virtual machines."
+    }.merge(@project_permissions.include?("Vm:create") ? {
       description: "Get started by creating a new virtual machine.",
       button_link: "#{@project_data[:path]}/vm/create",
       button_title: "New Virtual Machine"
-    }
+    } : {})
```

Comment:
without create button if no permission

<img width="1836" alt="Screenshot 2023-08-15 at 12 57 56" src="https://github.com/ubicloud/ubicloud/assets/993199/c94166c2-d430-442c-84af-7f2a04b0f0b2">

Reply:
Trying to learn this